DocumentedExamples/NearestNeighbourDistance.ipynb

```diff
@@ -0,0 +1,2113 @@
+{
```

Comment:
<p>OK, still I have a problem with the title. The "nearest neighbour distance from every cell in the grid" is 1 grid-cell length, isn't it?</p><p>I feel that the title is confusing because it blends some technical term "nearest neighbour" but uses that in the plain English sense? Or is it just me?</p><p> </p><p>Is this title better? <code>Calculating pairwise distances between a contour and every grid cell</code>?If so, then perhaps we can start with</p><p> </p><blockquote>In this example we will demonstrate how to compute the nearest distances of every grid cell to a particular contour. We will demonstrate this by calculating the closest distance from each grid cell in our data array to the sea ice edge. We will use monthly sea ice concentration from ACCESS-OM2-01 over the Southern Ocean. In order to do this efficiently we will use the haversine formula that determines the distance on the sphere between two locations on the sphere and the blah blah algorithm in order to pursue the computation efficiently.</blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Ok, on second thought, the last bit of this intro sentence is now included further down. So perhaps the first intro sentence could be just

> In this example we will demonstrate how to compute the nearest distances of every grid cell to a particular contour. We will demonstrate this by calculating the closest distance from each grid cell in our data array to the sea ice edge. We will use monthly sea ice concentration from ACCESS-OM2-01 over the Southern Ocean. 

?